packages/website/docs/using-vue.md

```diff
@@ -28,7 +34,9 @@ Begin by adding a container for your autocomplete menu. This example adds a `div
 </template>
 ```
 
-Then, import the necessary packages for a basic implementation. Since the example displays only recent searches, it imports the [`createLocalStorageRecentSearchesPlugin`](createlocalstoragerecentsearchesplugin) function from the [`autocomplete-plugin-recent-searches`](createlocalstoragerecentsearchesplugin) package. Depending on your desired [sources](sources) you may need to import other plugins and packages.
+Then, import the necessary packages for a basic implementation. Since the example queries an Algolia index, it imports the [`algoliasearch`](https://www.npmjs.com/package/algoliasearch) package and [`autocomplete`](autocomplete-js) and [`getAlgoliaHits`](getAlgoliaHits-js) from the [`autocomplete-js`](autocomplete-js) package. Finally it imports [`autocomplete-theme-classic`](autocomplete-theme-classic) package for some out of the box styling.
```

Comment:
```suggestion
Then, import the necessary packages for a basic implementation. Since the example queries an Algolia index, it imports the [`algoliasearch`](https://www.npmjs.com/package/algoliasearch) package, [`autocomplete`](autocomplete-js) and [`getAlgoliaHits`](getAlgoliaHits-js) from the [`autocomplete-js`](autocomplete-js) package. Finally, it imports [`autocomplete-theme-classic`](autocomplete-theme-classic) package for some out of the box styling.
```